### PR TITLE
fix: add missing major IDs

### PR DESCRIPTION
**Pull request by DonkeyBlaster**
https://discord.com/channels/942560349817831435/942564261287313459/1051657453802098688

### CODE REVIEW

File: app/Http/Enums/MajorIdentifications.php [hunk 1]
```diff
@@ -29,6 +29,8 @@ enum MajorIdentifications
    case EXPLOSIVE_IMPACT;
    case GEOCENTRISM;
    case GRAVITYWELL;
+   case DESC_SNOWYSTEPS;
```
Comment (DevChromium) [inline app/Http/Enums/MajorIdentifications.php:32]: Shouldn't this  be `SNOWY_STEPS`

File: app/Http/Enums/MajorIdentifications.php [hunk 2]
```diff
@@ -29,6 +29,8 @@ enum MajorIdentifications
...
+   case DESC_FESTIVESPIRIT;
```
Comment (DevChromium) [inline app/Http/Enums/MajorIdentifications.php:33]: Same here but `FESTIVE_SPIRIT`

File: app/Http/Enums/MajorIdentifications.php (hunk 1 above)
Comment (DonkeyBlaster) [reply app/Http/Enums/MajorIdentifications.php:32]: image

Doesn't look like it

File: app/Http/Enums/MajorIdentifications.php (hunk 2 above)
Comment (DonkeyBlaster) [reply app/Http/Enums/MajorIdentifications.php:33]: image

File: app/Http/Enums/MajorIdentifications.php (hunk 1 above)
Comment (DevChromium) [reply app/Http/Enums/MajorIdentifications.php:32]: Alright, thanks for clarifying!

**Review by DevChromium — APPROVED** [review]

**Review by DevScyu — APPROVED** [review]

**Pull request merged**